lib/tooling_invoker/invoke_locally.rb

```diff
@@ -31,7 +31,12 @@ def call
         cmd: cmd,
         exit_status: exit_status
       }
-      job.result = File.read("#{output_dir}/#{job.results_filepath}")
+
+      job.output_files = {}
+      job.output_filepaths.each do |output_filepath|
+        job.output_files[output_filepath] = File.read("#{output_dir.job_dir}/#{output_filepath}")
+      end
```

Comment:
Slight suggested refactor:

```suggestion
      job.output_files = job.output_filepaths.each.with_object({}) do |output_filepath, hash|
        hash[output_filepath] = File.read("#{output_dir.job_dir}/#{output_filepath}")
      end
```